charts/tenant-base/chart/templates/configmap.yaml

```
@@ -3,7 +3,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ printf "%s-%s" (include "chart.fullname" $) .name }}
+  name: {{ .name }}
```

We should do the same here and while we are it also in `persistantVolumeClaim.yaml` I guess?
```suggestion
  name: {{ printf "%s-%s" $.Release.Name $name }}
```

---

You are right